frontend/public/components/overview/project-overview.tsx

```diff
@@ -210,18 +209,34 @@ const projectOverviewListItemDispatchToProps = (dispatch): ProjectOverviewListIt
 const ProjectOverviewListItem = connect<ProjectOverviewListItemPropsFromState, ProjectOverviewListItemPropsFromDispatch, ProjectOverviewListItemOwnProps>(projectOverviewListItemStateToProps, projectOverviewListItemDispatchToProps)(
   ({dismissDetails, item, metrics, selectItem, selectedUID}: ProjectOverviewListItemProps) => {
     const {current, obj} = item;
-    const {namespace, name, uid} = obj.metadata;
+    const {name, uid} = obj.metadata;
     const {kind} = obj;
     // Hide metrics when a selection is active.
     const hasSelection = !!selectedUID;
     const isSelected = uid === selectedUID;
     const className = classnames(`project-overview__item project-overview__item--${kind}`, {'project-overview__item--selected': isSelected});
+    const handleClick = () => {
+      if (isSelected) {
+        dismissDetails();
+      } else {
+        selectItem(uid);
+      }
+    }
+    const onClick = (e: Event) => {
+      e.preventDefault();
+      handleClick();
+    };
+    // patternfly expects function with the signature of
+    // Type '(e: Event) => void' is not assignable to type '(event: MouseEvent<HTMLAnchorElement>) => void'.
```

Comment:
We can cast or change the signature. We should ideally have only one click handler.